draft-ftbs-rats-msg-wrap.md

```diff
@@ -45,21 +45,57 @@ normative:
 
 informative:
   I-D.ietf-rats-architecture: rats-arch
+  I-D.ietf-rats-eat: rats-eat
+  I-D.ietf-rats-ar4si: rats-ar4si
+  I-D.fossati-tls-attestation: tls-a
+  DICE-arch:
+    author:
+      org: "Trusted Computing Group"
+    title: "DICE Attestation Architecture"
+    target: https://trustedcomputinggroup.org/wp-content/uploads/DICE-Attestation-Architecture-r23-final.pdf
+    date: March, 2021
 
 --- abstract
 
 This document defines two encapsulation formats for RATS conceptual
-messages.
+messages (e.g., evidence, attestation results, endorsements and
+reference values.)
 
 --- middle
 
 # Introduction
 
+The RATS architecture defines a handful of conceptual messages
+({{Section 8 of -rats-arch}}). Each conceptual message can have multiple
+serialization formats ({{Section 9 of -rats-arch}}). The same serialized
+message may have to be transported via different protocols - for
+example, EAT {{-rats-eat}} evidence in a "background check" topological
+arrangement, AR4SI {{-rats-ar4si}} attestation results in "passport"
+mode.
+
+In order to minimize the cost associated with registration and maximize
+interoperability, it is desirable to reuse their typing information
+across such boundaries.
+
 This document defines two encapsulation formats for RATS conceptual
-messages ({{Section 8 of -rats-arch}}).
+messages that aim to achieve the goals stated above.
+
+These encapsulation formats are designed to be:
```

Comment:
Wording should say "there are two formats" rather than "designed to have two formats"

Reply:
I don't understand the editorial suggestion.  It doesn't seem to apply to L83.

Reply:
Sounds like the suggested change would be along the lines of
```suggestion
There are two encapsulation formats available:
```

Reply:
yes, but that doesn't work with what follows: the bulleted list describes the design criteria of the encaps, not the encaps themselves.

Reply:
Ah, right, I see now!

Reply:
closing this and merging.